.prettierrc.js

```diff
@@ -1,4 +1,4 @@
 module.exports = {
     tabWidth: 4,
-    printWidth: 90
+    printWidth: 90,
```

Comment:
Prettier running on itself lol

Reply:
Am I running the prettier wrong? I use the command "npx prettier --write . "

Reply:
Are you running it while in the same directory as this file? Regardless, PR looks good now so it's approved